src/pretix/base/models/event.py

```diff
@@ -1641,10 +1641,18 @@ class EventMetaProperty(LoggedModel):
         help_text=_("If checked, an event can only be taken live if the property is set. In event series, its always "
                     "optional to set a value for individual dates")
     )
-    allowed_values = models.TextField(
+    allowed_values = models.JSONField(
         null=True, blank=True,
+        encoder=CustomJSONEncoder,
         verbose_name=_("Valid values"),
-        help_text=_("If you keep this empty, any value is allowed. Otherwise, enter one possible value per line.")
+    )
+    filter_public = models.BooleanField(
```

Comment:
Do we need a way to sort the MetaProperties? It might be overkill as I guess usually one will only filter by one or two properties. But then, when it is two, maybe one wants to change the order around?